packages/jest-worker/src/index.js

```diff
@@ -67,13 +67,16 @@ export default class {
     }
 
     // Build the options once for all workers to avoid allocating extra objects.
```

Comment:
Good to keep all shared props at a common object, but probably the comment does not apply anymore, since we need to create one object per worker now.

Reply:
agree